src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+body {
+  margin: 0;
+  display: block;
+  --main-color: black;
+  --active-color: #00acdc;
+}
+
+html {
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-family: "Roboto", sans-serif;
+}
+
+.header__container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14px;
+  letter-spacing: 0;
+  text-align: left;
+  text-transform: uppercase;
+}
+
+.navigation {
+  margin: 0;
+  padding: 0;
```

Comment:
![image](https://user-images.githubusercontent.com/86923089/202899347-5495604f-fa8a-44eb-a133-6aac78a2a06b.png)

make sure you've deployed the latest changes and tests is up-to-date as well